src/documentation/text/02_Theory/index.tex

```diff
@@ -0,0 +1,19 @@
+Aktoren stellen nebenläufige Einheiten dar, welche nicht über einen gemeinsamen Speicherbereich verfügen, sondern über Nachrichten kommunizieren (Message passing).
+Die Kommunikation basiert auf dem Versenden von Nachrichten zu Empfängern (FIFO-Prinzip).
+Jeder Aktor verfügt dabei über einen Posteingang, eine Adresse und ein Verhalten.
+Nach erhalt einer Nachricht können Aktoren mit drei verschiedenen Funktionen reagieren:
+Nachrichten an sich selbst oder andere Aktoren verschicken, neue Aktoren erzeugen oder das eigene Verhalten ändern.
+Der Nachrichtenaustausch erfolgt asynchron, was bedeutet, dass der Sender nach versenden einer Nachricht sofort mit einer anderen Aktion fortfahren kann.\ifootcite{doc.akka.io_actorsintro}
+
+% TODO: add figure 
+
+Aktoren werden überwiegend bei funktionalen Programmiersprachen eingesetzt.
+In der Programmiersprache Erlang stellt es die Basis der Nebenläufigkeit dar.
+Aktoren können aber auch in imperativen Programmiersprachen, wie in C++ durch die Theron"=Bibliothek, implementiert werden.
+
+Da Aktoren zu einem Zeitpunkt nur eine Nachricht abarbeiten und kein anderer paralleler Prozess den internen Speicherbereich eines Aktors beeinflusst, wird die Fehleranfälligkeit verringert.
+Werte behalten über die gesamte Bearbeitung der Nachricht hin ihre Gültigkeit, was das Aktorenmodell grundlegend von klassischen Shared"=Memory"=Concurrency"=Ansätzen unterscheidet, bei denen alle nebenläufigen Threads auf einen gemeinsamen Speicherbereich zugreifen und durch Locking geschützt wird.
```

Comment:
`Werte behalten über die gesamte Bearbeitung der Nachricht hin ihre Gültigkeit` keine Ahnung was damit gemeint sein soll. Was du in diesem Absatz sagen willst, ist ja, dass keine Inkonsistenzen auftreten können, weil eine Nachricht nach der anderen abgearbeitet wird und nachrichten readonly sind. Den Absatz sollte man mMn umformulieren....